sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/flink-functions/arithmetic-functions.sqrl

```diff
@@ -0,0 +1,87 @@
+IMPORT flink.*; -- Import flink functions
+IMPORT data-local.Numbers; -- Import boolean data
+
+
+/* Create numeric data source */
+UnmodifiedNumericData := SELECT id,
+                                abs(avg(x)) AS x,
+                                abs(avg(y)) / 100 AS y,
+                                avg(z) AS z
+                         FROM Numbers GROUP BY id ORDER BY id;
+
+
+/* =======TEST CASES======== */
+
+/*+test */
+OperatorsTest := SELECT id,
+                         +x AS positive_operator,
+                         -x AS negative_operator,
+                         x + y AS addition_operator,
+                         x - y AS subtractions_operator,
+                         x * y AS multiplication_operator,
+                         x / y AS division_operator
+                         /*x % y AS mod_operator,*/
+                  FROM UnmodifiedNumericData
+                  ORDER BY id;
+
+/*+test */
+FunctionsTest := SELECT id,
+                        POWER(x, 3) AS power_function,
+                        ABS(x) AS absolute_function,
+                        /*SQRT(x) AS square_root_function,*/
```

Comment:
This is an issue with the parser, the workaround is:

    `SQRT`(x) AS square_root_function,

Reply:
I'll leave it like this until the parser is fixed. The goal is for the function to work sans workarounds.

Reply:
Thanks, created #997